modules/export_pipeline/spatialize_chunk_predictions.tf

```diff
@@ -93,8 +93,9 @@ resource "google_cloudfunctions2_function" "spatialize_chunk_predictions_functio
   }
 
   service_config {
-    available_memory    = "4Gi"
+    available_memory    = "32Gi"
```

Comment:
I'd like to find a middle ground between 4 & 32
Hopefully you can measure usage at 32 and find what you actually need here?